application/models/User_model.php

```diff
@@ -241,7 +241,8 @@ function update_session($id) {
 			'user_eqsl_qth_nickname' => $u->row()->user_eqsl_qth_nickname,
 			'user_hash'		 => $this->_hash($u->row()->user_id."-".$u->row()->user_type),
 			'radio' => isset($_COOKIE["radio"])?$_COOKIE["radio"]:"",
-			'station_profile_id' => isset($_COOKIE["station_profile_id"])?$_COOKIE["station_profile_id"]:""
+			//'station_profile_id' => isset($_COOKIE["station_profile_id"])?$_COOKIE["station_profile_id"]:""
+		    'station_profile_id' => $u->row()->user_station_id
```

Comment:
Minor: indent

Reply:
Hello Maescool,

thanks for reviewing my code, and many thanks for your comments!
First of all, sorry for the wrong indentation - that's something I hadn't noticed because I'm developing with Eclipse, and Eclipse doesn't make a difference between spaces and tabs. And since I'm normally using spaces while Peter uses tab stops it looked okay in my editor view (and still does). I will fix this, and use tabs for indentation from now on.
Concerning the logbook code for the station profile, you're right, the model is the right place for this - this was my first approach, and since it worked I haven't spend more time to design rules, which is of course something that should be fixed.
Also, the submit event in the onchange handler was my first (working) idea, I'm sure there's a better solution for this. 
But, as said, it's only a first approach, and I thought it would be good to upload it to see what other developers who are a bit more familiar with the code think about it so they can tell me if I'm on the right way.
And, as we see, this has happened, and it seems that I'm not completely wrong, :-).
Let's see what Peter thinks about my approach, his first comments gave me a lot of stuff to think about, :-).